File: IE 211 Fall21/src/Lab1/Dog.java
package Lab1;

public class Dog extends Animal{
	
	private String genre;
	
	public Dog() {}
	
	public void setGenre(String genreIn)
	{
		genre = genreIn;
	}

	public void isGenre(String genreIn)
	{
		if(genre == genreIn)
			System.out.println("The genre of the dog is " + genreIn);
		else 
			System.out.println("The genre of the dog is not " + genreIn);
	}
	
	public void speak()
	{
		System.out.println("Hav hav!");
	}
}

File: IE211 Fall22/src/Lab1/Dog.py
from Lab1.Animal import Animal

class Dog(Animal):
    __genre = None

    def __init__(self):
        pass

    def setGenre(self, genre):
        self.__genre = genre

    def isGenre(self, genre):
        if self.__genre == genre:
            print(f"The genre of the dog is {genre}")
        else:
            print(f"The genre of the dog is not {genre}")

    def speak(self):
        print("Hav hav :)")